types/src/Experiment/enums.ts

```diff
@@ -44,6 +44,12 @@ export enum SERVER_ERROR {
   CONDITION_NOT_FOUND = 'Condition not found',
 }
 
+export enum MARKED_DECISION_POINT_STATUS {
+  APPLIED = 'applied',
+  NOT_APPLIED = 'not-applied',
+  MARKED = 'marked',
```

Comment:
I had to go back and read the discussion in #275 to remember what the difference is between `MARKED` and `APPLIED`.  To me, that indicates these are poorly named -- or at the very least need some JSDoc to tell future developers what they mean.

What about `CONDITION_APPLIED`, `CONDITION_FAILED_TO_APPLY`, and `NO_CONDITION_ASSIGNED`?  Is that too verbose?

Reply:
This looks good to me. I will update the code accordingly